src/thermo/RedlichKwongMFTP.cpp

```diff
@@ -463,7 +463,7 @@ void RedlichKwongMFTP::initThermoXML(XML_Node& phaseNode, const std::string& id)
                 if (!isnan(coeffArray[0])) {
                     //Assuming no temperature dependence (i,e a1 = 0)
                     setSpeciesCoeffs(iName, coeffArray[0], 0.0, coeffArray[1]);
-                    m_coeffs_from_db[i] = true;
+                    m_coeffSource[i] = CoeffSource::EoS;
```

Comment:
Just FYI, codecov complains that this 'if' branch is not covered by a test, but that seems ok, given that it follows the exact same logic seen elsewhere, which is covered.

Reply:
Yeah, I think I don't want to worry about test coverage in this function, since this is just the deprecated XML-based initialization anyway. A lot versions of `initThermoXML` no longer have test coverage as we've migrated most of the input files used for testing to YAML.